scripts/rollup/bundles.js

```diff
@@ -169,19 +169,24 @@ const bundles = [
       sourceMap: false,
     },
     entry: 'src/renderers/dom/ReactDOMServer.js',
-    externals: ['prop-types', 'prop-types/checkPropTypes'],
+    externals: [
+      'prop-types',
+      'prop-types/checkPropTypes',
+      'create-react-class/factory',
+    ],
     fbEntry: 'src/renderers/dom/ReactDOMServer.js',
     hasteName: 'ReactDOMServerStack',
     isRenderer: true,
     label: 'dom-server',
     manglePropertiesOnProd: false,
     name: 'react-dom/server',
     paths: [
+      'src/isomorphic/**/*.js',
```

Comment:
Oops. These changes are actually pretty bad. We're building all of isomorphic into these packages now.